src/wp-includes/block-template-utils.php

```diff
@@ -1492,7 +1492,7 @@ function inject_ignored_hooked_blocks_metadata_attributes( $changes, $request )
 		return $changes;
 	}
 
-	$terms = $changes->tax_input;
+	$terms = isset( $changes->tax_input ) ? $changes->tax_input : array();
```

Comment:
Would we want to fallback to `array( 'wp_theme' => get_stylesheet() )` I wonder?

Reply:
The empty array will return an error since `wp_theme` wont be present. I think that is fine rather than for us to make assumptions in the code, if you agree.

Reply:
Agree!